R/migrate.R

```diff
@@ -63,10 +63,6 @@ migrate <- function(
   project <- renv_project_resolve(project)
   renv_project_lock(project = project)
 
-  # for jetpack
-  if (renv_package_checking())
-    renv_patch_repos()
```

Comment:
This doesn't do anything because in `renv_patch_repos()`:

https://github.com/rstudio/renv/blob/8bff10a3cca3ad1eeecf43ae3ee5f0b6f9dbe44b/R/patch.R#L168-L171

Reply:
I don't understand; I think the conditionals do match up here? This was to workaround a bug in the jetpack package which caused `renv` CRAN submissions to fail, so you probably want to be careful about modifying this code (it could break revdeps)

Reply:
Oh doh, I was just confused. I'll just remove the conditionals instead.

Reply:
Oh actually, I don't think these are needed any more since `renv_patch_repos()` is called from `renv_patch_init()` which is called from `renv_zzz_load()` which is called `.onLoad()`.